                                                                                                                                                              ERROR:daisy.client:Block downscale/1839 with read ROI [6400:9600, 9216:10240, 10240:10592] (3200, 1024, 352) and write ROI [6400:9600, 9216:10240, 10240:10592] (3200, 1024, 352) failed in worker 1976
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
downscale ▶:   0%|                                                                                    | 0/484 [00:01<?, ?blocks/s, ⧗=0, ▶=13, ✔=0, ✗=270, ∅=0]                                                                                                                                                              ERROR:daisy.client:Block downscale/196 with read ROI [9600:9700, 4096:5120, 2048:3072] (100, 1024, 1024) and write ROI [9600:9700, 4096:5120, 2048:3072] (100, 1024, 1024) failed in worker 1976
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
downscale ▶:   0%|                                                                                    | 0/484 [00:01<?, ?blocks/s, ⧗=0, ▶=13, ✔=0, ✗=270, ∅=0]                                                                                                                                                              ERROR:daisy.client:Block downscale/1411 with read ROI [9600:9700, 9216:10240, 7168:8192] (100, 1024, 1024) and write ROI [9600:9700, 9216:10240, 7168:8192] (100, 1024, 1024) failed in worker 1976
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
downscale ▶:   0%|                                                                                    | 0/484 [00:01<?, ?blocks/s, ⧗=0, ▶=13, ✔=0, ✗=270, ∅=0]